mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -910,8 +907,8 @@ impl NftListTable {
 impl TableSignature for NftListTable {
     const TABLE_NAME: &'static str = "nft_list_cache_table";
 
-    fn on_upgrade_needed(upgrader: &DbUpgrader, old_version: u32, _new_version: u32) -> OnUpgradeResult<()> {
-        if is_initial_upgrade(old_version) {
+    fn on_upgrade_needed(upgrader: &DbUpgrader, old_version: u32, new_version: u32) -> OnUpgradeResult<()> {
+        if old_version == 0 && new_version == 2 {
```

Comment:
this check is repeated at most  3 times. Can you add a helper fn like this in this mod
```rust
pub fn is_v2_upgrade(old_version: u32, new_version: u32) -> bool { old_version == 0 && new_version == 2 }
`

Reply:
pls check the last changes, you currently referencing to outdated code